types/Tree.d.ts

```diff
@@ -1,9 +1,15 @@
 import * as React from 'react';
 import { TreeBaseProps } from './TreeBase';
 
+interface DataItem {
+  value: any;
+  label: React.ReactNode;
+  children?: Array<DataItem>;
```

Comment:
Array type using 'Array<T>' is forbidden for simple types. Use 'T[]' instead

https://travis-ci.org/rsuite/rsuite/jobs/455358070#L521